source/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,124 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+You can query data based on geographical location using
+`geospatial query operators
+<https://docs.mongodb.com/manual/geospatial-queries/index.html>`_.
+Geospatial queries require a geospatial index.
+
+Overview
+--------
+
+Geospatial queries work with two different formats depending on your use
+case.
+
+Coordinates on an Earth-like Sphere
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For geospatial queries using longitude and latitude coordinates
+on an Earth-like sphere, use the `GeoJSON
+<https://docs.mongodb.com/manual/geospatial-queries/index.html#geospatial-geojson>`_
+format. While GeoJSON has `multiple types
+<https://docs.mongodb.com/manual/reference/geojson/>`_, GeoJSON data
+generally looks like this:
+
+.. code-block:: javascript
+
+   <field> : {
+      type: <GeoJSON type>,
+      coordinates: [
+         [longitude1, latitude1],
+         ...
+         [longitudeN, latitudeN]
+      ]
+   }
+
+The number of coordinates depends upon the object type; a Point requires
+only one coordinate: a longitude and a latitude. A Line requires
+two coordinates: a longitude and a latitude for each end. Polygons
+consist of a list of coordinates in which the first and last coordinate
+are the same, effectively closing the polygon. To learn more about
+the GeoJSON shapes you can use in MongoDB, consult the
+`GeoJSON manual entry
+<https://docs.mongodb.com/manual/reference/geojson/>`_. To geospatially
+query GeoJSON data, you must first add your GeoJSON data to a
```

Comment:
It seems like this is applicable for all the different types of 2D queries listed here (or at least the first two).
Would this be something you can link to? Or perhaps be added to its own sub-section?

Reply:
You mean creating the index? GeoJSON data uses a `2dsphere index` whereas coordinate pairs use a `2d` index. I want to keep them separate here because I think they two different kinds of index can be confusing when presented together.